catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -125,6 +129,18 @@
   private static final Supplier<Boolean> ZERO_PAGESIZE_COMPATIBILTY =
       () -> Boolean.valueOf(System.getProperty(ZERO_PAGESIZE_COMPATIBILITY_PROPERTY));
 
+  @VisibleForTesting
+  static final String SOLR_COMMIT_NRT_METACARDTYPES = "solr.commit.nrt.metacardTypes";
+
+  private final Set<String> commitNrtMetacardType =
```

Comment:
:question: why not make it static?

Reply:
I tried that first.  The test will fail if it is static since the class is loaded before the System property is set.  It is effectively final since this class is only created one by blueprint.

Reply:
I noticed after. I was forced to do the same and for the same reasons.